packages/extension/src/companion/index.tsx

```diff
@@ -9,18 +9,29 @@ import { setOnError } from '@dailydotdev/shared/src/components/ProfilePicture';
 import { getCompanionWrapper } from '@dailydotdev/shared/src/lib/extension';
 import App, { CompanionData } from './App';
 
+let root: ReturnType<typeof createRoot>;
```

Comment:
Is there no other way to solve this then with let?
EG just declaring as root const?

Reply:
We can't unfortunately. At initial load, the companion wrapper is not there yet for the root to reference. So when the time we need to render, the root is empty.